internal/azure.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute"
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork"
+	"github.com/Naman1997/discovr/verbose"
```

Comment:
This file still imports "log" and "fmt" packages. Make sure only the verbose package is used, otherwise it means that we have missed some lines where we are logging stuff.